vsg/styles/minimal.yaml

```diff
@@ -0,0 +1,9 @@
+# The name must be a single word
+name : minimal
+skip_phase:
+  - 1
+  - 2
+  - 3
+  - 5
+  - 6
+  - 7
```

Comment:
This can be written in following way `skip_phase: [1, 2, 3, 5, 6, 7]`

Reply:
thx, I will make the change.

Reply:
I just pushed an update.